cmd/storage-rest-server.go

```diff
@@ -137,17 +138,29 @@ func (s *storageRESTServer) CrawlAndGetDataUsageHandler(w http.ResponseWriter, r
 		return
 	}
 
-	w.Header().Set(xhttp.ContentType, "text/event-stream")
-	doneCh := sendWhiteSpaceToHTTPResponse(w)
-	usageInfo, err := s.storage.CrawlAndGetDataUsage(GlobalServiceDoneCh)
-	<-doneCh
-
+	w.Header().Set(xhttp.ContentType, "application/binary")
```

Comment:
We cannot change the contentType here because of requirements to let proxies know that they shouldn't buffer this request since its part of https://www.w3schools.com/html/html5_serversentevents.asp (server sent events) - it was on purpose we kept it `"text/event-stream"`

Reply:
If for some reason there are proxies between MinIO servers

Reply:
Reverted.